node/router/src/heartbeat.rs

```diff
@@ -314,7 +317,7 @@ pub trait Heartbeat<N: Network>: Outbound<N> {
             .iter()
             .filter_map(|peer_ip| {
                 // If the peer is not connected, attempt to connect to it.
```

Comment:
Change the comment to something like 
```suggestion
                // If the peer is not already connecting or connected, attempt to connect to it.
```